### PR TITLE
Fs 3273 output tracker

### PR DESCRIPTION
**Pull request by adamdavies1**
https://dluhcdigital.atlassian.net/browse/FS-3273

### Change description
Added a output tracker so user can run the report themselves. 

Iv tried to make this partially dynamic so eventually we could potentially pass in the field we want from a UI and run a report that way

to be used with https://github.com/communitiesuk/funding-service-design-assessment/pull/369

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
Click the new Assessment tracker export link on the landing page as highlighted from the picture

### Screenshots of UI changes (if applicable)
image

### CODE REVIEW

**Review by srh-sloan — CHANGES_REQUESTED** [review]
One comment on some logic - if it needs changing, there should really be a unit test to cover it.
Also, the unit tests seem quite light for what's a big amount of logic - we should be testing more branches in `get_export_data` I think.

File: db/queries/assessment_records/queries.py [hunk 1]
```diff
@@ -806,3 +848,26 @@ def add_missing_elements_with_empty_values(finalList):
             ordered_dict[key] = field.get(key, "")
         field.clear()
         field.update(ordered_dict)
+
+
+def combine_dicts(applications_list, scores_list):
+    combined_list = []
+
+    if len(applications_list) == 0 and len(scores_list) == 0:
+        return combine_dicts
```
[the remark this answers is not in the document]
Comment (adamdavies1) [reply db/queries/assessment_records/queries.py]: Good catch, weirdly this still worked

**Review by srh-sloan — APPROVED** [review]
Approved, but can you add a tech debt ticket to add more tests pls? Thansk :)

**Review by srh-sloan — APPROVED** [review]
Nice!

**Pull request merged**